addons/mk6mortar/XEH_initPost.sqf

```diff
@@ -0,0 +1,5 @@
+#include "script_component.hpp"
+
+// if (GVAR(useAmmoHandling)) then {
+    _this call FUNC(mortarInit);
+// };
```

Comment:
This GVAR isn't read from the module in time to work on preplaced mortars.
Any suggestions on how I can delay execution until the variable's been read?